staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -203,6 +203,9 @@ type DomainSpec struct {
 	// One of: shared, auto
 	// +optional
 	IOThreadsPolicy *IOThreadsPolicy `json:"ioThreadsPolicy,omitempty"`
+	// IOThreads specifies how many IOThreads are dedicated to the disks.
+	// +optional
+	IOThreads *DiskIOThreads `json:"ioThreads,omitempty"`
```

Comment:
Yeah fair, it is slightly confusing with `disk.dedicatedIOThread` also present in the API.

Reply:
Well, the IOThreads count is shared between all the disks which has `dedicatedIOThread` field enabled. It could be 3 disks but you want to request 4 IOthreads. Then, the load is spread over the 4 ioThreads.

Reply:
Ah sorry I misunderstood the meaning of that field, okay that makes sense and I can see why you think `ioThreadsPolicy` is redundant in that case. Definitely a candidate to be deprecated in `v1`.